ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -92,6 +96,14 @@ public final class SystemColumns {
           true,
           true,
           true
+      ),
+      PseudoColumn.of(
+          ROWID_NAME,
+          ROWID_TYPE,
+          ROWID_PSEUDOCOLUMN_VERSION,
+          true,
```

Comment:
Similar question here regarding materializing this column for table joins. Is it not possible to pull it out of the key in the existing state store?